http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/RuntimeRoutes.scala

```diff
@@ -201,13 +210,30 @@ class RuntimeRoutes(runtimeService: RuntimeService[IO], userInfoDirectives: User
       )
       _ <- runtimeService.startRuntime(userInfo, googleProject, runtimeName)
     } yield StatusCodes.Accepted
+
+  private[api] def updateRuntimeHandler(userInfo: UserInfo,
```

Comment:
I wonder if it makes sense to put `UserInfo`, `GoogleProject`, `RuntimeName` in `RuntimeServiceContext`

Reply:
Not all routes have `GoogleProject` and `RuntimeName` (e.g. list clusters)… also I feel like `UserInfo` is better left explicit.